stylesheet.css

```diff
@@ -14,7 +14,6 @@
 }
 
 .cosmic-app-scroll-view {
-	height: 504px;  /* 168 * 3 */
```

Comment:
Removing this causes the dialog to change height when switching to a folder with less than three rows of apps, which is undesirable.

Also after this, even with enough rows of apps it seems to not quite fit 3 rows (on my 1440p display); the labels are cut off.

Reply:
I'll take a look... I think the hardcoded numbers will not work for larger displays.